From 303d5e59a3000a44e4845f8dd73b0d764728803f Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Wed, 27 Mar 2013 17:48:44 +0100
Subject: [PATCH] WIP: bcm2835: Add Broadcom BCM2835 framebuffer to the device
 tree

This adds a device tree binding for framebuffer driven via mailbox IPC
mechanism present on Broadcom BCM2835 SoC, used in Raspberry Pi and Roku 2
devices.

Signed-off-by: Lubomir Rintel <lkundrak@v3.sk>
Signed-off-by: Dom Cobley <popcornmix@gmail.com>
Cc: Stephen Warren <swarren@wwwdotorg.org>
Cc: linux-rpi-kernel@lists.infradead.org
Cc: devicetree-discuss@lists.ozlabs.org
Cc: linux-arm-kernel@lists.infradead.org
---
 arch/arm/boot/dts/bcm2835.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/bcm2835.dtsi b/arch/arm/boot/dts/bcm2835.dtsi
index 853d6a2..f09d4e3 100644
--- a/arch/arm/boot/dts/bcm2835.dtsi
+++ b/arch/arm/boot/dts/bcm2835.dtsi
@@ -121,6 +121,11 @@
 		brcm,bcm2835-mbox = <&mailbox>;
 	};
 
+	fb {
+		compatible = "brcm,bcm2835-fb";
+		brcm,bcm2835-mbox = <&mailbox>;
+	};
+
 	clocks {
 		compatible = "simple-bus";
 		#address-cells = <1>;
-- 
1.8.4

